CodeEdit/Features/Settings/Pages/TextEditingSettings/Models/TextEditingSettings.swift

```diff
@@ -5,7 +5,7 @@
 //  Created by Nanashi Li on 2022/04/08.
 //
 
-import AppKit
+import AppKit.NSFont
```

Comment:
No need to import the specific NSFont module there's no benefit, remove this diff to keep the imports simpler.